nixos/modules/tasks/network-interfaces.nix

```diff
@@ -207,6 +207,15 @@ let
         type = with types; listOf (submodule (routeOpts 4));
         description = ''
           List of extra IPv4 static routes that will be assigned to the interface.
+          If the route type is the default <literal>unicast</literal>, then
```

Comment:
I would mention `networking.useNetworkd` explicitely, like:
```
If the route type is the default <literal>unicast</literal>, then the scope is
set differently depending on the value of <option>networking.useNetworkd</option>:
the script-based backend sets it to <literal>link</literal>, while networkd sets
it to <literal>global</literal>.
```
Also, it may be a good idea to put the paragraph in a `<warning></warning>` block.

Reply:
Yes, your version is better.
I added the `<warning></warning>` block.
I kept the paragraph "If you want consistency between the two implementations [...]". Do you think I should remove that too?

Reply:
No, I think it's fine this way. Thank you.